Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -75,10 +162,31 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```

Comment:
InsertLast should use InsertAt(array, number, array.Length-1)

Reply:
And the same for Remove

Reply:
InsertLast(int[] array, int number) now using InsertAt(array, number, array.Length)

RemoveLast(int[] array) now using RemoveAt(array, array.Length - 1);

RemoveFirst(int[] array) now using return RemoveAt(array, 0);